source/brailleDisplayDrivers/seikantk.py

```diff
@@ -212,39 +210,37 @@ def _processCommand(self, command: bytes, arg: bytes) -> None:
 			log.warning(f"Seika device has received an unknown command {command}")
 
 	def _handInfo(self, arg: bytes):
```

Comment:
While here let's rename this to `_handleInfo` prevent any confusion about hands.